### ISSUE TITLE
Combo boxes do not announce how many items or relationship

### ISSUE THREAD

**Issue by richnoah** [w3c/aria-at-app#429]
During testing conducted for the AT and Browser Capture feature, the following issue was discovered.  Comboboxes for browser and screen reader versions do not announce how many items there are or the relationship of the focused item compared to the overall number of items that can be selected.
